code-challenges.test.js

```diff
@@ -57,6 +127,50 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe("isFullHouse", ()=>{
+    test("checks if the array is fullhouse array or not", ()=>{
+        expect(isFullHouse([5, 5, 5, 3, 3])).toEqual(true)
+    })
+    test("checks if the array is is fullhouse array or not", ()=>{
+        expect(isFullHouse([5, 5, 3, 3, 4])).toEqual(false)
+    })
+    test("checks if the array is is fullhouse array or not", ()=>{
+        expect(isFullHouse([5, 5, 5, 5, 4])).toEqual(false)
+    })
+})
 
 
 // b) Create the function that makes the test pass.
+
+/*
+Algorithm-> chech if a given array is a fullHouse array
+Input->an array of five numbers
+Output=>boolean value that confirms if the array is fullHouse array or not.
+    create a function
+    declare an embpty array
+    declare an empty object
+    loop the array using for..of..loop
+    loop the counts object
+    push counts[key] to an empty array
+    loop over the new array
+    if conditionals
+    return
+*/
+function isFullHouse(array){
+    let newArr=[]
+const counts = {};
+for (const element of array) {
+  counts[element] = counts[element] ? counts[element] + 1 : 1;
+}
+for(const key in counts){
+    newArr.push(counts[key])
+}
+
+for(let i=0;i<newArr.length;i++){
+  if(newArr[i]===3)return true
+  
+}return false
+}
+//   console.log(isFullHouse(hand1))
+//   console.log(isFullHouse(hand2))
+//   console.log(isFullHouse(hand3))
```

Comment:
Besure to be using Jest whenever you can be. I know using the node system to log the results is easier-- but getting used to seeing testing logs will be more valuable to you in the long run!